src/renderer/qt/qt_window_manager.cc

```diff
@@ -341,16 +341,61 @@ void FillCandidates(const commands::Candidates &candidates,
 }
 }  // namespace
 
+struct VirtualRect {
+  const QScreen *screen;
```

Comment:
Please follow https://google.github.io/styleguide/cppguide.html#Declaration_Order

The data members should come last.